### PR TITLE
new: add xlsform conversion service; configure backend for it.

### PR DESCRIPTION
**Pull request by issa-tseng**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -45,6 +45,9 @@ services:
       - "443:443"
     healthcheck:
       test: [ "CMD-SHELL", "nc -z localhost 443 || exit 1" ]
+  xlsform:
+    container_name: xlsform
+    image: 'yanokwa/pyxform-http:v0.3.0'
```
Comment (yanokwa) [inline docker-compose.yml]: I'm assuming this is trivial to change if we move the repo.

Comment (issa-tseng) [reply docker-compose.yml]: yes

File: files/service/config.json.template [hunk 2]
```diff
@@ -14,6 +14,10 @@
         "port": 25
       }
     },
+    "xlsform": {
```
Comment (yanokwa) [inline files/service/config.json.template]: I think you missed this one?

Comment (issa-tseng) [reply files/service/config.json.template]: nope. that is the key central-backend wants to read, and i feel like in that case it's fine for it to generically be "xlsform"

**Pull request merged**